cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -373,52 +373,55 @@ void SparsePoseGraph::FreezeTrajectory(const int trajectory_id) {
   });
 }
 
-void SparsePoseGraph::AddSubmapFromProto(const int trajectory_id,
-                                         const transform::Rigid3d& initial_pose,
+void SparsePoseGraph::AddSubmapFromProto(const transform::Rigid3d& pose,
```

Comment:
Let's follow #591 and consistently name poses after the frame they relate to, i.e. here it would be `global_pose`.

Reply:
Done.